docker/Makefile.image_names

```diff
@@ -37,8 +37,9 @@ SERIALBOX_IMAGE ?= $(GCR_URL)/serialbox-build:$(TAG_NAME)
 ENVIRONMENT_IMAGE ?= $(GCR_URL)/fv3core-environment:$(TAG_NAME)
 
 # built image names
+FV3_TAG ?=
 FORTRAN_IMAGE = $(GCR_URL)/$(FORTRAN_NAME):$(TAG_NAME)
 WRAPPER_IMAGE = $(GCR_URL)/$(WRAPPER_NAME):$(TAG_NAME)
```

Comment:
These may also want the jenkins tag appended. If not, maybe a `#` comment as to why not.